zerver/views/webhooks/trello/board_actions.py

```diff
@@ -0,0 +1,107 @@
+from typing import Dict, Any, Tuple
+from .exceptions import UnknownUpdateBoardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_BOARD_ACTIONS = [
+    'removeMemberFromBoard',
+    'addMemberToBoard',
+    'createList',
+    'updateBoard',
+]
+
+REMOVE_MEMBER = 'removeMemberFromBoard'
+ADD_MEMBER = 'addMemberToBoard'
+CREATE_LIST = 'createList'
+CHANGE_NAME = 'changeName'
```

Comment:
Same suggestion as above, `action_type`s should be `text_type`. So prefix them with a `u`.